src/styles/style.css

```diff
@@ -0,0 +1,285 @@
+*, *::after, *::before{
```

Comment:
Do not style with HTML elements, it can mess up specificity of styles. Create a CSS class for each element.